frontend/src/metabase/home/components/EmbedHomepage/InteractiveTabContent.tsx

```diff
@@ -0,0 +1,82 @@
+// eslint-disable-next-line no-restricted-imports
+import { List } from "@mantine/core";
+import { Link } from "react-router";
+import { t, jt } from "ttag";
+
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { Anchor, Button, Icon, Text } from "metabase/ui";
+
+import type { EmbedHomepageViewProps } from "./EmbedHomepageView";
+
+export const InteractiveTabContent = (props: EmbedHomepageViewProps) => {
+  const {
+    embeddingAutoEnabled,
+    interactiveEmbeddingQuickstartUrl,
+    exampleDashboardId,
+    licenseActiveAtSetup,
+    learnMoreInteractiveEmbedUrl,
+  } = props;
+  return (
+    <>
+      <Text
+        mb="md"
+        lh="1.5"
+      >{t`Use interactive embedding to offer multi-tenant, self-service analytics and dashboard creation in their own data sandbox. Pro and Enterprise plans only. `}</Text>
+
+      <Text fw="bold" mb="md">{t`The TL;DR:`}</Text>
+
+      <List
```

Comment:
What are the reasons/benefits for using `List` here? I mean I understand Mantine has it, but this doesn't look anything different from using simple `ol` and `li` tags. If we need to use icons then maybe I could understand we might need `List` but that doesn't seem to be the case here.

Reply:
Yeah itt felt overkill to me too but right now we have a [reset that removes the list type](https://github.com/metabase/metabase/blob/49586158e6c6d607f89d410c5d5af2aaea4fa6e2/frontend/src/metabase/css/core/base.module.css#L44-L49), so we'd have to make a `styled(ol)`/css class with
```css
  list-style-type: decimal;
  list-style-position: inside;
 ```
I did it for the minimal embedding card but it felt like it should have been a re-usable component.

It should also help with making sure spacing, font sizes etc are consistent